tests/AssertTest.php

```diff
@@ -833,6 +833,20 @@ public function testResourceOfTypeCustomMessage(): void
 
         Assert::resource(null, 'curl', 'I want a resource of type %2$s. Got: %s');
     }
+
+    public function testEnumAssertionErrorMessage(): void
+    {
+        if (version_compare(PHP_VERSION, ENUM_INTRODUCTION_VERSION, '<')) {
+            $this->expectNotToPerformAssertions();
+
+            return;
```

Comment:
Other tests are using `PHP_VERSION_ID` for comparison and skip the test if necessary. Wouldn't that be simpler and more consistent?

Reply:
Ok, added VERSION_ID comparison instead of versions.